source/simulator/parameters.cc

```diff
@@ -611,6 +611,14 @@ namespace aspect
                          "as opposed to continuous. This then requires the assembly of face terms "
                          "between cells, and weak imposition of boundary terms for the composition "
                          "field via the discontinuous Galerkin method.");
+      prm.declare_entry ("Use limiter for discontinuous solution", "false",
+                         Patterns::Bool (),
+                         "Whether to apply the bound preserving limiter as a postprocess after having"
```

Comment:
Please add a space at the end of each line before the quotation mark (except the last line). The reason is this will be converted to a .tex document and used for the manual, and there you need the spaces between the sentences.